src/App.tsx

```diff
@@ -1,25 +1,48 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.css';
 import { getNumbers } from './utils';
+// import { it } from 'node:test';
+import { Pagination } from './components/Pagination';
 
-// eslint-disable-next-line @typescript-eslint/no-unused-vars
 const items = getNumbers(1, 42).map(n => `Item ${n}`);
 
 export const App: React.FC = () => {
+  // vars
+  const [sortViewCount, setSortViewCount] = useState('5');
+  const [currentPage, setCurrentPage] = useState(1);
+
+  // vars for items
+  const startIndex = (currentPage - 1) * Number(sortViewCount);
+  const endIndex = startIndex + Number(sortViewCount);
+  const currentItems = items.slice(startIndex, endIndex);
+
+  // method to change count of pages
+  const handlePageChange = (page: string) => {
+    setCurrentPage(1);
+    setSortViewCount(page);
+  };
```

Comment:
The `handlePageChange` function sets the current page to 1 every time the number of items per page is changed. This might not be the desired behavior if you want to maintain the current page after changing the items per page. Consider whether this logic aligns with your requirements.